### DISCUSSION TITLE
Heroku未來將停止免費服務預計11/28

### DISCUSSION THREAD

**Discussion by sogadisk** [libsgh/PanIndex#362] [Ideas]
Heroku未來將停止免費服務預計11/28
能否部屬到其他免費容器?

**Comment by libsgh** [comment]
可以看下这里：https://libsgh.github.io/PanIndex/#/quickstart?id=%e5%85%b6%e4%bb%96%e5%b9%b3%e5%8f%b0%e9%83%a8%e7%bd%b2
如果你还知道其他免费容器可以告知我
